src/anemoi/training/diagnostics/plots.py

```diff
@@ -351,6 +358,8 @@ def plot_predicted_multilevel_flat_sample(
         Expected data of shape (lat*lon, nvar*level)
     y_pred : np.ndarray
         Predicted data of shape (lat*lon, nvar*level)
+    precip_like_vars : set, optional
+        Set of precipitation-like variables, by default {"tp", "cp"}
```

Comment:
Do we want to keep this as default? just wondering if it might conflict when someone train using a dataset where there is not tp/cp defined

Reply:
Good question! If tp/cp aren't defined this default should have no effect. The function just tests if they are in the set {tp, cp} and if not it will use the default plotting. Do you think this is okay?